pimd/pim_oil.c

```diff
@@ -550,8 +551,14 @@ int pim_channel_add_oif(struct channel_oil *channel_oil, struct interface *oif,
 		return -4;
 	}
 
-	channel_oil->oil.mfcc_ttls[pim_ifp->mroute_vif_index] =
-		PIM_MROUTE_MIN_TTL;
+	/* This section is to avoid installation of iif in oif LIST
```

Comment:
How does this not break linux packet forwarding?  The ttl is used to decide if the ttl received for the packet is sufficient and the packet should be forwarded.